src/compositions/Questionnaire/Question/Question.js

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from 'react';
+import { ReactComponent as Arrow } from '../../../data/images/Arrow-Down-Right.svg';
+
+import './Question.css';
+
+import {
+    Date,
+    Radio,
+    RadioWithFollowUp,
+    Checkbox,
+    TextInput,
+    TextArea,
+    DropDown,
+    Email,
+    PhoneNumber,
+    Zip,
+} from '../../../components/FormComponents/FormComponents';
+
+const Question = ({ question, bindField, followUpQuestions }) => {
+    const [showFollowUp, setShowFollowUp] = useState(false);
+
+    const { text, required } = question;
+
+    const getInputType = (q) => {
+        switch (q.questionType) {
+            case 'date':
+                return (
+                    <Date
+                        slug={q.slug}
```

Comment:
why slug?

Reply:
"A slug is a human-readable, unique identifier, used to identify a resource instead of a less human-readable identifier like an id. You use a slug when you want to refer to an item while preserving the ability to see, at a glance, what the item is."(https://itnext.io/whats-a-slug-f7e74b6c23e0)

It's just a unique identifier, I can select all of them and rename them to ID if that makes it easier to understand